styles/mobile.css

```diff
@@ -3,6 +3,10 @@
         font-size: 0.7em;
     }
 
+    a:hover {
```

Comment:
:hover makes no sense on mobile

Reply:
Yep, curse of the copy paste. Fixed in 4af9f4d